tests/mod_kanban_boardmanager_test.php

```diff
@@ -0,0 +1,283 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace mod_kanban;
+
+/**
+ * Unit test for mod_kanban
+ *
+ * @package     mod_kanban
+ * @copyright   2021-2023, ISB Bayern
+ * @author      Stefan Hanauska <stefan.hanauska@csg-in.de>
+ * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ * @covers      \mod_kanban\boardmanager
+ */
+class mod_kanban_boardmanager_test extends \advanced_testcase {
```

Comment:
AFAIK the convention for the test class name would be: `nameoftheclasstobetested_test` without leading `mod_kanban`, because test classes now have a proper namespace (like you did, good job)